apps/meteor/client/views/admin/apps/AppSetting.tsx

```diff
@@ -90,7 +90,7 @@ function AppSetting({ appSetting, onChange, value, ...props }: AppSettingProps):
 			value={value}
 			onChangeValue={onChange}
 			_id={id}
-			{...{ translatedValues }}
+			{...{ values: translatedValues }}
```

Comment:
Not sure if your fix is valid but syntax wise, this is the same as `value={translatedValues}`. So no need to do a spread.

Reply:
Yups, makes sense :+1: Modified the spread condition to only add that field if it's not null. Can you please check it again?